docs/xss.rst

```diff
@@ -0,0 +1,95 @@
+Performing XSS Vulnerability Audits
+==================================
+
+The bok-choy framework includes the ability to perform XSS (cross-site scripting) audits on
+web pages using a short XSS locator defined in
+https://www.owasp.org/index.php/XSS_Filter_Evasion_Cheat_Sheet#XSS_Locator.
+
+Leverage Your Existing Tests and Fail on XSS Vulnerability Errors
+-------------------------------------------------------------
+
+To trigger XSS vulnerability audits passively, set the ``VERIFY_XSS``
+environment variable to ``True``. Doing so triggers an XSS audit
+whenever a page object's ``q`` method is called. If errors are
```

Comment:
I think this refers to the proof-of-concept implementation...since it's now on `wait_for_page`